presto-docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -20,6 +20,11 @@ properties in the file:
     connection-user=root
     connection-password=secret
 
+.. note::
```

Comment:
Maybe it is worth adding that it may have negative performance implications?
@electrum, @mosabua WDYT?

Reply:
Probably yes ... seems like a bad idea to implement this way to some degree but its a Oracle issue. So in the warning about the performance this should maybe link to the Oracle docs about the impact.

Reply:
Thanks for your feedback. Updated as follows:
<img width="776" alt="Screen Shot 2020-10-20 at 17 41 36" src="https://user-images.githubusercontent.com/6237050/96562560-ddab1480-12fb-11eb-86a5-4815004f1814.png">

Reply:
Nice. Thanks